### DISCUSSION TITLE
ibax cuzdanı kurtarma yöntemi kesin çözüm...

### DISCUSSION THREAD

**Discussion by yasin001yk** [IBAX-io/go-ibax#55] [Q&A] [answered]
arkadaşlar cuzdanla alakalı sorun yasayanlar bu dedıgım yolu ızlerlerse herkes cuzdanını kurtarabılır öncelikle yapmanız gereken çok basıt  cuzdanı kurarken verılen kurtarma kelimeleri elınızde olması gerekıyor ÖNCELIKLE YENI CUZDAN KUR KISMINA TIKLIYORUZ DAHA ONCE VERILEN KELIMELERI BURAYA YAPIŞTIRIYORUZ ESKI ŞİFREMİZİ TEKRAR GİRİYORUZ AYNI SEYLERİ İKİNCİ KEZ YINE YAPIP KURTARMA KELİMELERİ VE SİFREYI GİRİYORUZ bukadar basıt eskısı gıbı şifrenızı tekrar girip giriş yapabılırsınız private key olmadan kurtarma kelimleri yle cuzdanınızı kurtarma yontemı ben uyguladım bı kaç arkadas daha uyguladı oldu bu bol şans herkese...

**Comment by HalloKyle** [comment, marked as answer]
Hi， please check this url for solution：
https://github.com/IBAX-io/go-ibax/discussions/62

**Comment by MstfYtskn** [comment]
> arkadaşlar cuzdanla alakalı sorun yasayanlar bu dedıgım yolu ızlerlerse herkes cuzdanını kurtarabılır öncelikle yapmanız gereken çok basıt cuzdanı kurarken verılen kurtarma kelimeleri elınızde olması gerekıyor ÖNCELIKLE YENI CUZDAN KUR KISMINA TIKLIYORUZ DAHA ONCE VERILEN KELIMELERI BURAYA YAPIŞTIRIYORUZ ESKI ŞİFREMİZİ TEKRAR GİRİYORUZ AYNI SEYLERİ İKİNCİ KEZ YINE YAPIP KURTARMA KELİMELERİ VE SİFREYI GİRİYORUZ bukadar basıt eskısı gıbı şifrenızı tekrar girip giriş yapabılırsınız private key olmadan kurtarma kelimleri yle cuzdanınızı kurtarma yontemı ben uyguladım bı kaç arkadas daha uyguladı oldu bu bol şans herkese...
Başkalarını davet edinc ya da github,medium ,discord gibi platformlardan takip ettim ama puan vermedi. Bu konuda bilginiz var mı?

**Comment by hemresenol** [comment]
Bazen tarayıcıyla alakalı da sorunlar oluşabiliyor.Girip cıkınca geçmişi çerezleri silince düzeliyor.

**Comment by cihanaslan0** [comment]
👆
